apps/studio/components/interfaces/Database/Functions/FunctionsList/FunctionsList.tsx

```diff
@@ -51,16 +50,11 @@ const FunctionsList = ({
     router.push(url)
   }
 
-  const canCreateFunctions = useCheckPermissions(
+  const { can: canCreateFunctions } = useAsyncCheckProjectPermissions(
     PermissionAction.TENANT_SQL_ADMIN_WRITE,
     'functions'
   )
 
-  const { data: schemas } = useSchemasQuery({
```

Comment:
This is added to preload data for the `SchemaSelector`. Maybe remove the `const { data` and add a comment?